src/status_im2/contexts/chat/photo_selector/album_selector/view.cljs

```diff
@@ -51,22 +52,19 @@
   (str (:title item) index))
 
 (defn album-selector
-  [{:keys [on-scroll]}]
-  (rf/dispatch [:chat.ui/camera-roll-get-albums])
-  (fn [{:keys [scroll-enabled]}]
-    (let [albums         (rf/sub [:camera-roll/albums])
-          selected-album (or (rf/sub [:camera-roll/selected-album]) (i18n/label :t/recent))]
-      [rn/view {:style {:padding-top 20}}
-       [album-title false]
-       [gesture/section-list
-        {:data                           albums
-         :render-fn                      album
-         :render-data                    selected-album
-         :sections                       albums
-         :sticky-section-headers-enabled false
-         :render-section-header-fn       section-header
-         :style                          {:margin-top 12}
-         :content-container-style        {:padding-bottom 40}
-         :key-fn                         key-fn
-         :scroll-enabled                 @scroll-enabled
-         :on-scroll                      on-scroll}]])))
+  [{:keys [scroll-enabled on-scroll]} album? selected-album]
```

Comment:
why not `scroll-enabled?`, it's a boolean, right?

Reply:
cc @OmarBasem , yeah it should be `scroll-enabled?` i guess

Reply:
Yeah `scroll-enabled` is a boolean

Reply:
should be addressed separately